go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/IBM/ibm-csi-common
 go 1.16
 
 require (
-	github.com/IBM/ibmcloud-volume-interface v1.0.0-beta8
-	github.com/IBM/ibmcloud-volume-vpc v1.0.0-beta12
+	github.com/IBM/ibmcloud-volume-interface v1.0.1-0.20211126135815-30d8a848c0e1
```

Comment:
Update the version to release one once the interface changes are committed

Reply:
done